http.go

```diff
@@ -161,12 +161,24 @@ func timestampForRequest(r *http.Request) time.Time {
 	return time.Unix(int64(timestamp), 0)
 }
 
+type Clock interface {
+	Now() time.Time
+}
+
+type utcClock struct {
+}
+
+func (c *utcClock) Now() time.Time {
+	return time.Now().UTC()
+}
+
 type RasterHttpServer struct {
 	cache       *filecache.FileCache
 	rasterCache *RasterCache
 	ring        ringman.Ring
 	urlSecret   string
 	agent       *gorelic.Agent
+	clock       Clock
```

Comment:
I made this refactor to enable testing. We don't want tests which stop working after 16 hours...